venues/ICLR.cc/2019/Conference/webfield/bidWebfield.js

```diff
@@ -8,6 +8,21 @@ var BLIND_INVITATION_ID = CONFERENCE_ID + '/-/Blind_Submission';
 var USER_SCORES_INVITATION_ID = CONFERENCE_ID + '/-/User_Scores';
 var ADD_BID = CONFERENCE_ID + '/-/Add_Bid';
 var PAGE_SIZE = 1000;
+var SEARCH_KEYWORDS = [
```

Comment:
@mohituniyal could you paste the keywords here?

Reply:
Also can you change the text in the instructions?

```
<li>You may search for papers by keywords</li>\
```

and say:

```
In order to have a better experience bidding the papers we recommend you to filter the results by Subject Areas (list of most used keywords) and paper metadata using the search textbox
```

@mspector , please correct me the text.

Reply:
My version:

"For the best bidding experience it's recommended that you filter papers by Subject Area and search for key phrases in paper metadata using the search form"

Reply:
Thanks Zach!